auth-service/src/main/kotlin/uk/gov/communities/delta/auth/security/RateLimiting.kt

```diff
@@ -0,0 +1,35 @@
+package uk.gov.communities.delta.auth.security
+
+import io.ktor.server.application.*
+import io.ktor.server.plugins.*
+import io.ktor.server.plugins.forwardedheaders.*
+import io.ktor.server.plugins.ratelimit.*
+import org.slf4j.LoggerFactory
+import uk.gov.communities.delta.auth.config.Env
+import kotlin.time.Duration.Companion.minutes
+
+fun Application.configureRateLimiting() {
+    val logger = LoggerFactory.getLogger("Application.RateLimiting")
+    val rateLimit = Env.getRequiredOrDevFallback("AUTH_RATE_LIMIT", "10").toInt()
+
+    install(XForwardedHeaders) {
+        this.skipLastProxies(1)
+    }
+    install(RateLimit) {
+        // Currently only the login page is rate limited so a status 429 (Too many requests) sends the login page (with an error message)
+        register(RateLimitName("protectLogin")) {
```

Comment:
Minor, but would be nice to have "protectLogin" as a public variable (`const val`) so other files can import it rather than needing the same string